extra/test_release_tool.py

```diff
@@ -13,23 +13,40 @@
 #    limitations under the License.
 
 import os
-import sys
-import shutil
-import re
 import pathlib
+import re
+import shutil
+import sys
 from unittest.mock import patch
 
 import pytest
-
-from release_tool import main
-from release_tool import docker_compose_files_list
-from release_tool import Component
+import yaml
+from release_tool import Component, docker_compose_files_list, main
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 RELEASE_TOOL = os.path.join(THIS_DIR, "release_tool.py")
 INTEGRATION_DIR = os.path.normpath(os.path.join(THIS_DIR, ".."))
 
 
+@pytest.fixture(scope="session")
+def is_staging():
+    """Fixture to figure out if we are running the tests in staging branch
+
+    Inspect git-versions.yml and ensure that the core OS repositories that have
+    enterprise forks (i.e. deployments, inventory, and useradm) do not exist; as
+    they shall exist only in git-versions-enterprise.yml.
+    """
+
+    non_staging_components = [
+        "mender-deployments",
+        "mender-inventory",
+        "mender-useradm",
```

Comment:
Shouldn't workflows also be here?

Reply:
It _could_ be there. But there are two `workflows` Docker images (server and worker), so for simplicity I made the fixture to only look for the core ones. It doesn't really matter, it could look for only one of them...

Reply:
Right, ok, it doesn't really matter as long as there is at least one positive match. Never mind then.